seacatauth/openidconnect/handler/introspect.py

```diff
@@ -111,10 +111,21 @@ async def introspect(self, request):
 			L.log(asab.LOG_NOTICE, "Access token matched no active session.")
 			return asab.web.rest.json_response(request, {"active": False})
 
-		response_data = {"active": True}
-
-		# Add OPTIONAL user info data
-		response_data.update(**await self.OpenIdConnectService.build_userinfo(session))
+		user_info = await self.OpenIdConnectService.build_userinfo(session)
+		response_data = {
+			# REQUIRED
+			"active": True,
+			# OPTIONAL
+			"token_type": "access_token",
+			"client_id": user_info.get("azp"),
+			"exp": user_info.get("exp"),
+			"iat": user_info.get("iat"),
+			"sub": user_info.get("sub"),
+			"aud": user_info.get("aud"),
+			"iss": user_info.get("iss"),
+		}
+		if "preferred_username" in user_info:
+			response_data["username"] = user_info["preferred_username"]
```

Comment:
Seacat normalizes `username` field to `preferred_username` since that is a standard OpenID claim, while `username` is not. but I can add the fallback there just to be sure

Reply:
fair enough, wasnt aware that `username` is not standard one